axon_schema.go

```diff
@@ -23,20 +24,32 @@ var orphanSequences []string
 // for each table, maps column names to their postgres data types
 var columnTypes = (map[string]map[string]string{})
 
+func parseSemver(version string) (major, minor int, err error) {
+	reg, err := regexp.Compile(`^(\d+)\.(\d+)`)
```

Comment:
is this function used mostly for parsing postgres versions `major.minor` versions?

Reply:
Ought to work on anything implementing SemVer. https://semver.org/